packages/doenetml-worker-rust/lib-doenetml-core/src/core/document_model/document_model.rs

```diff
@@ -174,9 +175,26 @@ impl DocumentModel {
     ) -> impl Iterator<Item = PropPointer> {
         let local_prop_indices = {
             let document_structure = self.document_structure.borrow();
+
+            // Determine whether or not component is in a graph (i.e., has a graph ancestor).
+            // If so, we'll look for props that are rendered `in_graph` rather than `in_text`.
+            let in_graph = document_structure
+                .get_true_component_ancestors(component_idx)
```

Comment:
I don't recall what this function exactly does, but does the `_true` mean it ignores parents coming from `extend`? Please verify that this is actually the call we want.

Reply:
That's right. It gets the parent from the DAST and skips extra parents such as the component extended.

Reply:
But is that what we want? If we have a point in a graph and we extend that point outside of a graph, we want the one outside the graph to use the text renderer.

Reply:
That's right. That's what this should do, given that the "true" ancestors of the extended point will not include a graph. (In the dast, the extended point doesn't descend from a graph, though it does descend from a graph via its extended link from its source.)

I didn't rely on my understanding; I tested examples. I now made an automated test demonstrating this: `rendered_props_with_references()` inside the point tests. Let me know if there is another case I should consider.

Reply:
I understand now. We do want the true parent and not the inherited one! Tests give me confidence :-)